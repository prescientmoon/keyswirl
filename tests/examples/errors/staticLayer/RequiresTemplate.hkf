module exporting ()

alias foo = []
layer bar using foo: "a" -- foo is a chord, not a template!
