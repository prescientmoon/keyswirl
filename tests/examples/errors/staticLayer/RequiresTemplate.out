[error WrongType]: Cannot match types
     +--> errors.staticLayer.RequiresTemplate@4:17-4:20
     |
   4 | layer bar using foo: "a" -- foo is a chord, not a template!
     :       ^---      ^-- 
     :       |         `- Only values of type
     :       |              LayerTemplate
     :       |            can be used as a static layer template! This variable has type
     :       |              Sequence
     :       |            instead.
     :       `- while checking this static layer
-----+
